### ISSUE TITLE
Formatting issue in two-sided CV

### ISSUE THREAD

**Issue by wojtusch** [opieters/limecv#26]
I created a CV with two pages following the approach mentioned in issue #12. I stumbled over a formatting issue that occurs when the first environment on the first page has more or less entries than the first environment on the second page. In this case, the line of the second environment on the second page that connects the bullet points seems to start from the first entry of the first environment and not of the second environment. I tested this with v0.1.7 and v0.1.8.

![Output](https://user-images.githubusercontent.com/6815837/111000489-6cd7c800-8382-11eb-8576-a89d92faed32.png)

The following MWE produces the shown result on the second page. As soon as the third item in the first environment is included, the formatting of the second page is as expected. The order of environments does not seem to have any influence on this issue. I also tried it with custom environments.

```
% !TEX TS-program = pdflatex
% !TEX encoding = UTF-8 Unicode

\documentclass[a4paper]{limecv}
  
\usepackage[margin=\cvMargin,noheadfoot]{geometry}
\usepackage{mwe}
\cvSetLanguage{english}
  
\begin{document}

% Sidebar
\begin{cvSidebar}
\cvID{John}{Doe}{example-image-a}{position}
\end{cvSidebar}

% Main
\begin{cvMainContent}
\begin{cvExperience}
\cvItem{Item 1}
\cvItem{Item 2}
%\cvItem{Item 3}
\end{cvExperience}
\begin{cvEducation}    
\cvItem{Item 1}
\cvItem{Item 2}
\cvItem{Item 3}
\end{cvEducation}
\end{cvMainContent}
  
% New page
\clearpage
\setcounter{cv@itemPrev}{0}
\setcounter{cv@itemNext}{1}
\setcounter{cv@lastItem}{1}

% Sidebar
\begin{cvSidebar}
\cvID{John}{Doe}{example-image-a}{position}
\end{cvSidebar}

% Main
\begin{cvMainContent}
\begin{cvEducation}    
\cvItem{Item 1}
\cvItem{Item 2}
\cvItem{Item 3}
\end{cvEducation}
\begin{cvExperience}
\cvItem{Item 1}
\cvItem{Item 2}
\end{cvExperience}
\end{cvMainContent}

\end{document}
```

**Comment by wcnvessies** [comment]
I came across the same formatting issue when added multiple pages and new sections. But I found a solution added the following lines of code to the class file.

```
\setcounter{cv@itemPrev}{0} % added to rest counters
\setcounter{cv@itemNext}{1} % added to rest counters
\setcounter{cv@lastItem}{1} % added to rest counters
```

See example below where to add, repeat this also for cvExperience and your own created sections.

```
\NewDocumentEnvironment{cvEducation}{}{%
    \cv@Title{\pgfkeysvalueof{/@cv/names/education}}{\faGraduationCap}
    \setcounter{cv@itemPrev}{0} % added to rest counters
    \setcounter{cv@itemNext}{1} % added to rest counters
    \setcounter{cv@lastItem}{1} % added to rest counters
    \stepcounter{cv@itemPrev}
    \stepcounter{cv@itemNext}
    \cv@definecvItem
  }{%
    \cv@EndSectionDraw
    \stepcounter{cv@lastItem}
    \draw (item \the\value{cv@itemPrev}.south west)
      node (cv@last item \the\value{cv@lastItem}) {};
}
```

**Comment by opieters** [comment]
Thanks for pointing out this issue. I will fix this later this week.

**Comment by opieters** [comment]
Issue fixed on version 0.1.10.
